docs/understanding-the-magic.rst

```diff
@@ -52,6 +52,16 @@ This allows you to access the session within the context of an incoming
 HTTP request, but it will *not* allow you access it outside that
 context.
 
+State & Security
+================
+
+One of the key features of :attr:`~OAuth2ConsumerBlueprint.session` is that 
+the requests it generates use a `state` variable to ensure that the source
+of OAuth authorization callbacks is in fact your intended OAuth provider.
+By default, the state is a random 30-character string, as provided by
+:func:`~oauthlib.common.generate_token`. This protects your app against one
```

Comment:
Please remove the `~`, so that the rendered HTML shows `oauthlib.common.generate_token()` instead of just `generate_token()`. Also, you'll need to add `oauthlib` to the `intersphinx_mapping` variable in `docs/conf.py`, in order to make this cross-reference work properly. Here are [the docs for the `intersphinx` extension](http://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html).